src/components/inputs/TextInput.js

```diff
@@ -129,6 +129,8 @@ export default class TextInput extends BaseInput {
 
   constructor(props) {
     super(props);
+    
+    console.warn('uilib TextInput component will be deprecated soon, please use TextField instead');
```

Comment:
It's not really deprecated, no? We'll use it for the basic ui, so it will break only the ui, not the build.

Reply:
Of course, currently, we have duplicates, from now on we'll only work on `TextField` for fixes and I'll create a breaking change PR for the change of `TextInput`

once we'll have few breaking PRs we'll merge them and update major version